robot_calibration/include/robot_calibration/ceres/chain3d_to_mesh_error.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2018-2022 Michael Ferguson
+ * Copyright (C) 2015 Fetch Robotics Inc.
+ * Copyright (C) 2013-2014 Unbounded Robotics Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
no need to add description in here if you plan on adding a readme sometime in the future